charts/opensearch-cluster/templates/opensearch-cluster-cr.yaml

```diff
@@ -27,6 +27,27 @@ spec:
     imagePullSecrets:
       {{ toYaml .Values.opensearchCluster.initHelper.imagePullSecrets | nindent 6 }}
     {{- end }}
+    {{- if .Values.opensearchCluster.initHelper.resources }}
+    resources:
+      {{- if .Values.opensearchCluster.initHelper.resources.requests }}
+      requests:
+        {{- if .Values.opensearchCluster.initHelper.resources.requests.memory }}
+        memory: {{ .Values.opensearchCluster.initHelper.resources.requests.memory }}
+        {{- end }}
+        {{- if .Values.opensearchCluster.initHelper.resources.requests.cpu }}
+        cpu: {{ .Values.opensearchCluster.initHelper.resources.requests.cpu }}
+        {{- end }}
+      {{- end }}
+      {{- if .Values.opensearchCluster.initHelper.resources.limits }}
+      limits:
+        {{- if .Values.opensearchCluster.initHelper.resources.limits.memory }}
+        memory: {{ .Values.opensearchCluster.initHelper.resources.limits.memory }}
+        {{- end }}
+        {{- if .Values.opensearchCluster.initHelper.resources.limits.cpu }}
+        cpu: {{ .Values.opensearchCluster.initHelper.resources.limits.cpu }}
+        {{- end }}
+      {{- end }}
+    {{- end }}
```

Comment:
This could be simplified like this:
```
{{- if .Values.opensearchCluster.initHelper.resources }}
resources:
  {{- .Values.opensearchCluster.initHelper.resources | nindent 6 }}
{{- end }}
```

Reply:
> Also, I know you only added the resources part, but could you please add the entire initHelper struct to the values.yaml of the chart to show that it is possible to set it?

Sure! I will include the other parameters.

Reply:
> This could be simplified like this:
Ok! I will simplify that part.